web_company_color/models/res_company.py

```diff
@@ -200,21 +201,27 @@ def _scss_get_sanitized_values(self):
         )
         return values
 
-    def _scss_generate_content(self):
+    def _scss_generate_content(self, uninstall=False):
         self.ensure_one()
         # ir.attachment need files with content to work
         if not self.company_colors:
             return "// No Web Company Color SCSS Content\n"
-        return self.SCSS_TEMPLATE % self._scss_get_sanitized_values()
+        if uninstall:
+            return (
+                self._get_scss_template_uninstall() % self._scss_get_sanitized_values()
```

Comment:
**suggestion:** now that `_get_scss_template_uninstall` doesn't actually exist, you could remove it (also from the method's parameters) and check for the context variable instead

Reply:
I missed to push the changes for scss_generate as testing was handled in different repo.

Not checking context variable inside _scss_generate_content and it is checked only inside _get_scss_template alone